tests/sentry/incidents/test_logic.py

```diff
@@ -2414,17 +2414,6 @@ def test_create_custom_metric_alert_rule(self, mocked_schedule_invalidate_projec
 
         mocked_schedule_invalidate_project_config.reset_mock()
 
-        with self.feature({"organizations:on-demand-metrics-prefill": True}):
```

Comment:
Maybe this whole test does not make sense anymore. Please let me know.

Reply:
The test itself is ok, i am just confused by the flag being set to true, it should have nothing to do with custom metrics. Can you try keeping test without the feature enabled. Thanks!